trojsten/fixtures/sites.json

```diff
@@ -70,5 +70,13 @@
         },
         "model": "sites.site",
         "pk": 10
+    },
+    {
+        "fields": {
+            "domain": "susi.sk",
```

Comment:
Toto je v poriadku? Neviem čo má robiť ten súbor

Reply:
Toto sa da naloadovat ako pociatocna data do db. Ale je lepsie tam mat spravnu hodnotu, tak to opravim.